ts/components/ContextualHelpModal.tsx

```diff
@@ -182,10 +184,11 @@ const ContextualHelpModal: React.FunctionComponent<Props> = (props: Props) => {
     >
       <Container>
         {showSendPersonalInfo ? (
-          <SendSupportTokenInfo
+          <SendSupportRequestOptions
+            initialScreenshotCheckboxValue={false}
```

Comment:
```suggestion
```
For now it's better to keep the default behavior: don't ask about screenshot sharing.